.circleci/config.yml

```diff
@@ -1239,6 +1239,10 @@ jobs:
             echo '76a04354dba9980fcbc35bece2957b30 geth-alltools-linux-amd64-1.13.14-2bd6bd01.tar.gz' | md5sum -c -
             tar -xzvf geth-alltools-linux-amd64-1.13.14-2bd6bd01.tar.gz
             sudo cp geth-alltools-linux-amd64-1.13.14-2bd6bd01/* /usr/local/bin
+      - run:
+          name: Install eth2-testnet-genesis
+          command: |
+            go install -v github.com/protolambda/eth2-testnet-genesis@v0.10.0
```

Comment:
This doesn't use `versions.json` to ensure the correct version is used

Reply:
`versions.json` isn't used at all in the circleci config, see e.g. geth, which also has a hardcoded version. So I just followed this pattern. Would you still prefer to use the `versions.json` here?

Reply:
OK added this, it works. Added a follow up ticket (https://github.com/ethereum-optimism/optimism/issues/10398) to use `versions.json` more consistently in the cci config.